widget/playground/src/utils/export.ts

```diff
@@ -114,6 +120,15 @@ export function formatConfig(config: WidgetConfig) {
     `,
     formatedConfig.indexOf('apiKey')
   );
+  if (config.trezorManifest) {
+    formatedConfig = insertAt(
+      formatedConfig,
+      `// Trezor Connect Manifest requires that you,
```

Comment:
The other three comments which are added to the result config require an action (like changing a value) but this comment is just giving some information about that field. Maybe it can be removed.